R/utils_gcp.R

```diff
@@ -29,9 +29,11 @@ gcp_gcs_bucket <- function() {
 gcp_gcs_head <- function(
   key,
   bucket = gcp_gcs_bucket(),
-  version = NULL
+  version = NULL,
+  quiet = TRUE
 ) {
-  suppressMessages(
+  loud <- if(!quiet) function(x){x} else suppressMessages
```

Comment:
One alternative is `if_any(quiet, suppressMessages, identity)`.

https://github.com/ropensci/targets/blob/f020d476c065abd686ffa2afad180f4735b6c024/R/utils_logic.R#L33-L39